cloud/aws/templates/aws_oidc/bin/aws_cli.py

```diff
@@ -37,11 +41,86 @@ def update_master_password_in_database(self, db_name: str, password: str):
             f"rds modify-db-instance --db-instance-identifier={db_name} --master-user-password={password} "
         )
 
-    def restart_ecs_service(self, cluster: str, service_name: str):
+    def restart_ecs_service(self):
+        """
+        Restarts the CiviForm ECS service.
+        """
         self._call_cli(
-            f"ecs update-service --force-new-deployment --service={service_name} --cluster={cluster}"
+            f"ecs update-service --force-new-deployment --cluster={self._ecs_cluster} --service={self._ecs_service}"
         )
 
+    def wait_for_ecs_service_healthy(self):
+        """
+        Polls the CiviForm ECS service, waiting for the PRIMARY deployment to
+        have rolloutStatus of COMPLETED.
+
+        Gives up after 20 tries, sleeps 30 seconds between each try.
+        """
+        print(
+            "\nWaiting for CiviForm ECS service to become healthy.\n"
+            f"Service URL: {self._get_url_of_ecs_service()}")
+
+        tries = 20
+        while True:
+            state = self._ecs_service_state()
+            if state == "COMPLETED":
+                print("Service is healthy.")
+                return
+
+            tries -= 1
+            if tries == 0:
+                print(
```

Comment:
Currently there are quite a few python files that print to stdout.  I think it would be better to keep the printing consistent until we update all the stdout usages to stderr.  I created https://github.com/civiform/civiform/issues/3967 to track moving to stderr.

Reply:
![image](https://user-images.githubusercontent.com/473671/208539043-0a6540ff-0925-4425-bb51-6489e7f044b6.png)